tasks/cleanup-workspace/cleanup-workspace.yaml

```diff
@@ -45,11 +45,11 @@ spec:
         fi
 
         echo "Delaying execution by $(params.delay) seconds"
-        sleep $(params.delay)
+        sleep "$(params.delay)"
 
         CLEANUP_DIR="$(workspaces.input.path)/$(params.subdirectory)"
 
         if [ -d "${CLEANUP_DIR}" ] ; then
           # Delete a directory and its contents
-          rm -rf "${CLEANUP_DIR}"/
+          rm -rf "${CLEANUP_DIR:?}/"
```

Comment:
Shellcheck probably complained that if the variable is empty, you will end up deleting `/`, right? How about just removing the trailing `/`? It seems redundant.

Reply:
OK. Updated. Thanks